@Czaki the test failures look real [here](https://github.com/napari/napari/actions/runs/7248838877/job/19745574907?pr=6540#step:11:489):

```
          # Uses something other than `self` the first arg to allow "self" as a settable attribute
          values, fields_set, validation_error = validate_model(__pydantic_self__.__class__, data)
          if validation_error:
  >           raise validation_error
  E           pydantic.v1.error_wrappers.ValidationError: 2 validation errors for LabelColormap
  E           colors
  E             Only up to 2**16=65535 colors are supported for LabelColormap (type=value_error)
  E           controls
  E             Wrong number of control points provided. Expected 1, got 131074 (type=value_error)
```

Can you fix? 🙏

--- a/napari/layers/labels/_tests/test_labels.py
+++ b/napari/layers/labels/_tests/test_labels.py
@@ -273,10 +273,14 @@ def test_num_colors():
     layer = Labels(data, num_colors=60)
     assert layer.num_colors == 60
 
-    with pytest.raises(ValueError, match="Because of implementation details"):
+    with pytest.raises(
+        ValueError, match=r".*Only up to 2\*\*16=65535 colors are supported"
+    ):
         layer.num_colors = 2**17
 
-    with pytest.raises(ValueError, match="Because of implementation details"):
+    with pytest.raises(
+        ValueError, match=r".*Only up to 2\*\*16=65535 colors are supported"
+    ):
         Labels(data, num_colors=2**17)
 
 

